src/python/pants/option/options_bootstrapper_test.py

```diff
@@ -142,67 +162,66 @@ def test_bootstrapped_options_ignore_irrelevant_env(self):
     self.assertIn(included, bootstrapper.env)
     self.assertNotIn(excluded, bootstrapper.env)
 
-  def do_test_create_bootstrapped_multiple_config(self, create_options_bootstrapper):
```

Comment:
This was inlined as it was only used by one test. Also, this implementation had a lot of duplication. That was reworked through the helper function `assert_config_read_correctly()`.